src/backend/jag-efiling-api/src/main/java/ca/bc/gov/open/jagefilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -70,7 +71,9 @@ public SubmissionApiDelegateImpl(
 
         logger.debug("Attempting to get user cso account information");
         CsoAccountDetails csoAccountDetails = efilingAccountService.getAccountDetails(generateUrlRequest.getUserId());
-
+        if (csoAccountDetails == null || !csoAccountDetails.HasRole(EFILING_ROLE)) {
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
we actually want to return success response here, the idea is to store if the user exist or not so we can route to the form on the front end

Reply:
Ahhh the ticket says return error?

Reply:
Ok I think I get the intent. Code updated